app/customer.py

```diff
@@ -0,0 +1,40 @@
+from dataclasses import dataclass
+import math
+
+from app.car import Car
+from app.shop import Shop
+
+
+@dataclass
+class Customer:
+    name: str
+    products_cart: dict
+    location: list[int]
+    money: int | float
+    car: Car

```

Comment:
Ensure that the `Car` class in `app.car` has all the necessary attributes and methods required by the `Customer` class, such as `fuel_consumption`.